platform/authorized_api_guide.md

```diff
@@ -20,15 +21,57 @@ API credentials which then allows you to request for a token that is used for au
   Name must contains at least 3 characters.
 * Copy your newly generated `CLIENT ID` and `CLIENT SECRET` because they **won't be available for you after dismissing this window.**
 
-### Create access token
+#### Create access token
 
-Having generated your API Credentials, now you are ready for creating access token that will be used in communication with API.
-To do so, send POST request 
+Having generated your API Credentials, now you are ready for creating access token that will be used in 
+communication with API.
+
+Piwik PRO API tokens use [JWT](https://jwt.io/) format.
+
+Make POST call to `https://<domain>/auth/token` with header `Content-Type: application/json` and payload: 
+`{ grant_type: 'client credentials', client_id: '<clientId>', client_secret: '<clientSecret>' }`.
+
+Response example:
 ```
-POST /auth/token
+{"token_type":"Bearer","expires_in":86400,"access_token":"<your_access_token>"}
 ```
 
-Curl example:
+Now, you can use obtained `<access_token>` for communication with Piwik PRO API.
+Field `expires_in` stands for time (in seconds) for token expiration (TTL).
+As token is a Bearer type, it must be **included in every API call** within header.
+
+```
+Authorization: Bearer <your-token-here>
```

Comment:
to be consistent: `<access_token>` (we've got `access_token`, `your_access_token` and `token` - in FAQ)?

Reply:
also maybe we could stick to one naming convention? we've got snake and camel cases here (`clientId` vs `access_token`)

Reply:
Rewritten to `your_access_token`, camel case reduces as much as possible